app/Http/Transformers/PhotoTransformer.php

```diff
@@ -36,8 +36,6 @@ public function transform(Photo $photo)
                     'url' => $photo->file_url,
                     'edited_url' => $photo->edited_file_url,
                 ],
-                'caption' => $photo->caption,
```

Comment:
@DFurnes yea..oops..didn't mean to delete this but it also seems like the photo transformer isn't being used anymore. but regardless, it will be removed when we move everything to the posts table.